package.json

```
@@ -9,13 +9,17 @@
       "import": "./lib/index.js",
       "require": "./dist/eslintrc.cjs"
     },
-    "./package.json": "./package.json"
+    "./package.json": "./package.json",
+    "./universal": {
+      "import": "./lib/index-universal.js",
+      "require": "./dist/eslintrc-universal.cjs"
+    }
   },
   "files": [
     "lib",
     "conf",
     "LICENSE",
-    "dist/eslintrc.cjs"
+    "dist"
```

```suggestion
    "dist/eslintrc.cjs",
    "dist/eslintrc-universal.cjs"
```

if we don't want to publish `.map` files

---

I can't think of a reason not to publish map files.

---

I don't know, we are not publishing map files in other dual packages e.g., https://github.com/eslint/eslint-visitor-keys/pull/24#discussion_r657121846